sinks.go

```diff
@@ -44,6 +46,25 @@ func (p *Pipe) CountLines() (int, error) {
 	return lines, p.Error()
 }
 
+// SHA256Sum create the SHA-256 of the file from the pipe's reader, and returns the
```

Comment:
Methods in each of these files are kept in alphabetical order, just to make things a little saner; see the [contributor's guide](https://github.com/bitfield/script/blob/master/CONTRIBUTING.md#style-and-formatting).

Reply:
The method just after is called `Stdout` so I think `SHA256` should be before this one.

Reply:
Sorry, I misread this; ignore me!